CalculatorTests/CalculatorTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,320 @@
+            List<string> actualMonths = new List<string>();
+            IWebElement startMonthField = driver.FindElement(By.Id("month"));
+            
+
+            SelectElement startMonthDropdown = new SelectElement(startMonthField);
+            foreach (IWebElement element in startMonthDropdown.Options)
+            {
+                actualMonths.Add(element.Text);
+            }
+            Assert.AreEqual(expectedMonths, actualMonths);
+           //* Assert.AreEqual(12, startMonthDropdown.Options.Count);
+        }
+
+
+        [Test]
+        public void JuneMonthDisplayBugTest()
+        {
+            //arrange
+            IWebElement monthDropdown = driver.FindElement(By.Id("month"));
+            IWebElement juneMonth = monthDropdown.FindElements(By.TagName("option"))[5];
+
+            //act
+            monthDropdown.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+            .Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementIsVisible(By.Id("month")));
+
+            //assert
+            Assert.AreEqual("June", juneMonth.Text);
+
+        }
+
+        [Test]
+        public void FinancialYearButtonMandatoryBugTest()
+        {
+            //arrange
+            IWebElement depositAmountField = driver.FindElement(By.Id("amount"));
+            IWebElement rateOfInterestField = driver.FindElement(By.Id("percent"));
+            IWebElement investmentTermField = driver.FindElement(By.Id("term"));
+            IWebElement calculateButton = driver.FindElement(By.Id("calculateBtn"));
+
+            //act
+            depositAmountField.SendKeys("100");
+            rateOfInterestField.SendKeys("100");
+            investmentTermField.SendKeys("100");
+
+            //assert
+            Assert.IsFalse(calculateButton.Enabled);
+        }
+
+        [Test]
+        public void EndDateBugTest()
+        { //arrange
+            IWebElement depositAmountField = driver.FindElement(By.Id("amount"));
+            IWebElement rateOfInterestField = driver.FindElement(By.Id("percent"));
+            IWebElement investmentTermField = driver.FindElement(By.Id("term"));
+            IWebElement startDayField = driver.FindElement(By.Id("day"));
+            IWebElement startMonthField = driver.FindElement(By.Id("month"));
+            IWebElement startYearField = driver.FindElement(By.Id("year"));
+            IWebElement financialYearButton365 = driver.FindElement(By.XPath("//tr[@id = 'finYear']/td[text() = '365 days']/*"));
+            IWebElement calculateButton = driver.FindElement(By.Id("calculateBtn"));
+            IWebElement interestEarnedField = driver.FindElement(By.Id("interest"));
+            IWebElement incomeField = driver.FindElement(By.Id("income"));
+            IWebElement endDateField = driver.FindElement(By.Id("endDate"));
```

Comment:
Please update the locators to find the inputs via their labels (using xpath) and delete obsolete tests: DepositAmountFieldBugTest, InterestEarnedOutputBugTest, IncomeOutputBugTest